src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
```

Comment:
The code correctly uses destructuring for accessing the `person` prop, which aligns with the checklist requirement to use destructuring for props.